x/tx/signing/get_signers.go

```diff
@@ -54,7 +54,7 @@ type getSignersFunc func(proto.Message) []string
 
 func getSignersFieldNames(descriptor protoreflect.MessageDescriptor) ([]string, error) {
 	signersFields := proto.GetExtension(descriptor.Options(), msgv1.E_Signer).([]string)
-	if signersFields == nil || len(signersFields) == 0 {
+	if len(signersFields) == 0 {
```

Comment:
Shouldn't we add an `signersFields, ok` and check the ok value as well? `if !ok || len(signersFields) == 0` in that case.

Reply:
I don't think we need to check for nil, unless we do care about if the slice is nil which doesn't seem like the case here. I think we would only care if we are passing this to an encoder that might encode it as null/nill instead of an empty array

Reply:
so, I am honestly not sure here.  I will ask the hive mind.